pyvista/core/common.py

```diff
@@ -1035,3 +1049,70 @@ def find_closest_cell(self, point):
         locator.BuildLocator()
         closest_cells = np.array([locator.FindCell(node) for node in point])
         return int(closest_cells[0]) if len(closest_cells) == 1 else closest_cells
+
+    def cell_n_points(self, ind):
+        """Return the number of points in a cell.
+
+        Parameters
+        ----------
+        ind : int
+            Cell ID.
+
+        Returns
+        -------
+        int
+            Number of points in the cell.
+
+        """
```

Comment:
Would be helpful to have an example here to demonstrate this method in action.

Reply:
Ok. I'll do it.